src/views/layouts/layout.ejs

```diff
@@ -2,6 +2,7 @@
 <html lang="en" class="h-100">
   <head>
     <%- include('../partials/head'); %>
+    <%- include('../pages/index'); %>
```

Comment:
This reflects poor understanding of how page layouts are constructed.

A page is constructed using a layout not vice versa.